### PR TITLE
ATB-1524 | Adding in SubscriptionEndpoint URL

### PR DESCRIPTION
**Pull request by MinaMahdi**
### What? 

Adding in the Integration URL for Pager Duty into our template 

### Why? 

To set up Pager Duty Integration for Production (manual deployment)

### CODE REVIEW

File: ais-infra-alerting/template.yaml [hunk 1]
```diff
@@ -42,7 +42,7 @@ Parameters:
     Description: >
       External API subscription endpoint i.e. PagerDuty events integration API
     Type: String
-    Default: "none"
+    Default: "https://events.pagerduty.com/integration/96e06442f974460fd0dd0b45a26c8f52/enqueue"
```
Comment (domantas-gds) [inline ais-infra-alerting/template.yaml]: Should be none by default else you're integrating in all environments

Comment (domantas-gds) [reply ais-infra-alerting/template.yaml]: So either a manual step to update parameter in Prod or remove this parameter, set a condition on parameter references to point to none unless `isProduction` then can point to actual PagerDuty endpoint.

Comment (domantas-gds) [reply ais-infra-alerting/template.yaml]: I'd go with second option, removes a manual step

Comment (MinaMahdi) [reply ais-infra-alerting/template.yaml]: Cool gotcha thanks 👍

Comment (domantas-gds) [reply ais-infra-alerting/template.yaml]: Actually, let's confirm we're fine to provide the PagerDuty endpoint in our code.

Comment (domantas-gds) [reply ais-infra-alerting/template.yaml]: Else will have to keep as-is and provide it manually in the existing CFN parameter

Comment (MinaMahdi) [reply ais-infra-alerting/template.yaml]: Spoke to someone from Security called Matthew and he said its okay to add the URL in the repos as its _not reachable from the internet_, so I guess its okay to keep in.

File: ais-infra-alerting/template.yaml [hunk 2]
```diff
@@ -55,11 +48,6 @@ Parameters:
       - "No"
 
 Conditions:
-  CreateSNSSubscription:
-    Fn::Not:
-      - Fn::Equals:
-          - !Ref SubscriptionEndpoint
-          - "none"
   IsNotDevelopment: !Not [ !Equals [ !Ref Environment, dev ]]
```
Comment (domantas-gds) [inline ais-infra-alerting/template.yaml:51]: Can remove this, only need isProd if only targeting production environment.

**Review by domantas-gds — APPROVED** [review]

**Review by TaherKapasi — APPROVED** [review]

**Pull request merged**